src/components/icon/styles.css

```diff
@@ -12,36 +27,73 @@
 	.icon.size_96x96::before {
 		display: block;
 		content: '';
-		width: 96px;
-		height: 96px;
 		top: 0;
 		left: 0;
+		bottom: 0;
+		right: 0;
 		position: absolute;
 		background-color: #f4f9fb;
 		border-radius: 48px;
 		z-index: -1;
 		}
 
+		.icon.size_96x96:hover:before {
+			top: -1px;
+			bottom: -1px;
+			left: -1px;
+			right: -1px;
```

Comment:
What's the reason behind negative values for top/bottom/left/right here?

Reply:
to adjust the size to the CSS box with 1px border